instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/package.py

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 
-_instruments = ("aiopg >= 0.13.0",)
+_instruments = ("aiopg <= 1.2.1",)
```

Comment:
shouldn't this be `>=0.13.0, <1.3.0` consider 1.3 introduced the breaking change and for some reason we did not want to support versions older than 0.13?

Reply:
I wasn't aware of the support of multiple conditions in python.
I have updated accordingly.

Reply:
Thanks. Can you please open an issue that tracks adding support for 1.3.0 and newer versions? Please also include details about the issue this instrumentation has today.

Reply:
Created